### PR TITLE
SEOImages: User Story 1463880, Part 16

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031600060/Report/

File: docs/framework/wpf/advanced/optimizing-performance-text.md [hunk 1]
```diff
@@ -127,8 +127,7 @@ ms.assetid: 66b1b9a7-8618-48db-b616-c57ea4327b98
 ### Showing Underlines on Hyperlinks Only on MouseEnter Events  
  A <xref:System.Windows.TextDecoration> object is a visual ornamentation that you can add to text; however, it can be performance intensive to instantiate. If you make extensive use of <xref:System.Windows.Documents.Hyperlink> elements, consider showing an underline only when triggering an event, such as the <xref:System.Windows.ContentElement.MouseEnter> event. For more information, see [Specify Whether a Hyperlink is Underlined](how-to-specify-whether-a-hyperlink-is-underlined.md).  
   
- ![Hyperlinks displaying TextDecorations](./media/textdecoration03.png "TextDecoration03")  
-Hyperlink appearing on MouseEnter
```
Comment (mairaw) [inline docs/framework/wpf/advanced/optimizing-performance-text.md]: consider moving this as an intro to the image

File: docs/framework/wpf/advanced/how-to-specify-whether-a-hyperlink-is-underlined.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ The <xref:System.Windows.Documents.Hyperlink> object is an inline-level flow con
   
  In the following example, the underline for the "My MSN" link is dynamic—it only appears when the <xref:System.Windows.ContentElement.MouseEnter> event is triggered.
```
Comment (mairaw) [inline docs/framework/wpf/advanced/how-to-specify-whether-a-hyperlink-is-underlined.md]:
```suggestion
 In the following example, the underline for the "My MSN" link is dynamic, that is, it only appears when the <xref:System.Windows.ContentElement.MouseEnter> event is triggered. 
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Great changes. Left just a few comments.

**Comment by damabe** [comment]
@mairaw Thanks! The changes are made. New CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031900035/Report/

**Review by mairaw — APPROVED** [review]
Looks great @damabe! Thanks! :shipit:

**Pull request merged**